src/features/contactmoment/ContactmomentVragenMenu.vue

```diff
@@ -70,31 +72,27 @@ async function waitForConfirmation() {
     display: flex;
     align-items: stretch;
     aspect-ratio: 1;
-
-    &:last-of-type {
-      grid-column: -2 / -1;
-    }
+    font-weight: bold;
   }
 
   button,
```

Comment:
I think we're setting (or rather trying to unset) a lot of the button values here, because a button comes with a lot of standard styling.

For this, you can use: `all: unset;`.

Reply:
Thanks, good point!